src/test/util_tests.cpp

```diff
@@ -26,12 +26,13 @@
 
 #include <array>
 #include <cmath>
+#include <cstdint>
+#include <cstring>
```

Comment:
nit: `cstring` and `string` includes can just be dropped according to [IWYU](https://api.cirrus-ci.com/v1/task/4921334028828672/logs/ci.log)